src/util/TraceUtil.v3

```diff
@@ -34,6 +34,29 @@ component TraceUtil {
 		out.putc('s');
 		return length + 1;
 	}
+	def renderThrowable(out: TraceBuilder, thrown: Throwable) {
```

Comment:
I think `out` can be of type `StringBuilder` for a little more generality. Also, if it returns the `StringBuilder` then it will work with `%q` in string formatting.